app/policies/user_policy.rb

```diff
@@ -11,8 +11,12 @@ def create?
     is_admin? || is_himself?
   end
 
+  def update?
+    is_himself?
+  end
+
   def is_himself?
-    current_user == record
+    record == current_user
```

Comment:
Part of this PR uses this method and it's not working right, fixing it to make the PR work properly is part the PR.